CHANGES.rst

```diff
@@ -6,9 +6,13 @@ Other
 
 - Provide second-order polynomial transforms for NIRCam WFSS grisms. [#124]
 
+
 - Deprecate ``stdatamodels.jwst.datamodels.DataModel`` in favor of
   ``stdatamodels.jwst.datamodels.JwstDataModel``. [#160]
 
+- Add wavelength tables for NIRSpec Drizzle cubepars reference file [#157]
```

Comment:
The PR number here is wrong. Should be 162, not 157. That's why the CI check is failing.